arcgis-ios-sdk-samples/Geometry/Format coordinates/FormatCoordinates.storyboard

```diff
@@ -178,16 +47,182 @@
+                                                <textField opaque="NO" contentMode="scaleToFill" contentHorizontalAlignment="left" contentVerticalAlignment="center" placeholder="45.0000N 120.0000E" textAlignment="natural" adjustsFontForContentSizeCategory="YES" minimumFontSize="17" translatesAutoresizingMaskIntoConstraints="NO" id="Flo-zm-KFZ">
+                                                    <rect key="frame" x="16" y="31.5" width="343" height="0.0"/>
+                                                    <nil key="textColor"/>
+                                                    <fontDescription key="fontDescription" style="UICTFontTextStyleBody"/>
+                                                    <textInputTraits key="textInputTraits" returnKeyType="done"/>
+                                                    <connections>
+                                                        <action selector="textFieldAction:" destination="geL-f0-eoD" eventType="editingDidEnd" id="R60-c0-Ieu"/>
+                                                        <outlet property="delegate" destination="geL-f0-eoD" id="QlL-Gw-ly6"/>
+                                                    </connections>
+                                                </textField>
+                                                <label opaque="NO" userInteractionEnabled="NO" contentMode="left" horizontalHuggingPriority="251" verticalHuggingPriority="251" verticalCompressionResistancePriority="751" text="Decimal Degrees" textAlignment="natural" lineBreakMode="tailTruncation" baselineAdjustment="alignBaselines" minimumScaleFactor="0.5" adjustsLetterSpacingToFitWidth="YES" adjustsFontForContentSizeCategory="YES" translatesAutoresizingMaskIntoConstraints="NO" id="5aq-8r-2lB">
+                                                    <rect key="frame" x="16" y="12" width="343" height="15.5"/>
+                                                    <fontDescription key="fontDescription" style="UICTFontTextStyleSubhead"/>
+                                                    <color key="textColor" white="0.33333333333333331" alpha="1" colorSpace="custom" customColorSpace="genericGamma22GrayColorSpace"/>
+                                                    <nil key="highlightedColor"/>
+                                                </label>
+                                            </subviews>
+                                            <constraints>
+                                                <constraint firstItem="5aq-8r-2lB" firstAttribute="leading" secondItem="297-yI-a6o" secondAttribute="leadingMargin" id="NRZ-fA-Vrf"/>
+                                                <constraint firstItem="Flo-zm-KFZ" firstAttribute="top" secondItem="5aq-8r-2lB" secondAttribute="bottom" constant="4" id="ija-gt-iDX"/>
+                                                <constraint firstItem="Flo-zm-KFZ" firstAttribute="trailing" secondItem="5aq-8r-2lB" secondAttribute="trailing" id="kVf-8F-EHK"/>
```

Comment:
The trailing constraints of each text field should be equal to the trailing margin of the super superview.